src/ipv6rwc/ipv6rwc.go

```diff
@@ -223,14 +206,17 @@ func (k *keyStore) oobHandler(fromKey, toKey ed25519.PublicKey, data []byte) { /
 }
 */
 
-/*
 func (k *keyStore) sendKeyLookup(partial ed25519.PublicKey) {
-	sig := ed25519.Sign(k.core.PrivateKey(), partial[:])
-	bs := append([]byte{typeKeyLookup}, sig...)
-	//_ = k.core.SendOutOfBand(partial, bs)
-	_ = bs
+	/*
+		sig := ed25519.Sign(k.core.PrivateKey(), partial[:])
```

Comment:
Can probably delete the commented out lines in various places if they're no longer useful.

Reply:
I think I'll leave it for the moment, and we can do a cleanup pass before we release.